app/src/main/kotlin/io/eugenethedev/taigamobile/ui/screens/commontask/components/CommonTaskAttachments.kt

```diff
@@ -24,34 +24,35 @@ import io.eugenethedev.taigamobile.domain.entities.Attachment
 import io.eugenethedev.taigamobile.ui.components.dialogs.ConfirmActionDialog
 import io.eugenethedev.taigamobile.ui.components.loaders.DotsLoader
 import io.eugenethedev.taigamobile.ui.components.texts.SectionTitle
-import io.eugenethedev.taigamobile.ui.screens.commontask.EditActions
+import io.eugenethedev.taigamobile.ui.screens.commontask.EditAction
 import io.eugenethedev.taigamobile.ui.screens.main.LocalFilePicker
 import io.eugenethedev.taigamobile.ui.utils.activity
+import java.io.InputStream
 
 @Suppress("FunctionName")
 fun LazyListScope.CommonTaskAttachments(
     attachments: List<Attachment>,
-    editActions: EditActions
+    editAttachments: EditAction<Pair<String, InputStream>, Attachment>
```

Comment:
Because I need to push it [here](https://github.com/EugeneTheDev/TaigaMobile/blob/c5454798eaf24e5caa2a783ff34ea801df45cb86/app/src/main/kotlin/io/eugenethedev/taigamobile/ui/screens/wiki/page/WikiPageScreen.kt#L185), and I don't want to create all EditActions, no?